provider/aws/cloudformation.go

```diff
@@ -345,6 +366,15 @@ func buildStack(stackDetails *cloudformation.Stack) *common.Stack {
 	} else {
 		stack.LastUpdateTime = aws.TimeValue(stackDetails.CreationTime)
 	}
+	// parse Region and Account out of ARN
+	arnChunks := strings.Split(stack.ID, ":")
```

Comment:
doesn't seem these are used...is this old?

Reply:
True, they're not used.  I removed this bit, and the assignment of AccountID and Region just below.